turbolinks/build.gradle

```diff
@@ -17,7 +17,7 @@ android {
     buildToolsVersion "23.0.2"
 
     defaultConfig {
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
I meant to, and did, lower it back down to 19. About 1/3 of users are on 19 last I checked.